src/main/webapp/app/shared/metis/post/post.component.ts

```diff
@@ -16,6 +16,8 @@ export class PostComponent extends PostingDirective<Post> implements OnInit, OnC
     // if the post is previewed in the create/edit modal,
     // we need to pass the ref in order to close it when navigating to the previewed post via post title
     @Input() modalRef?: NgbModalRef;
+    @Input() courseMessagesPageFlag: boolean;
```

Comment:
Probably rename to something like `isCourseMessagesPage`? The "flag" is duplicated since it's a boolean value.